lib/api/objectPutPart.js

```diff
@@ -374,6 +374,7 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
             newByteLength: size,
             oldByteLength: prevObjectSize,
             location: destinationBucket.getLocationConstraint(),
+            numberOfObjects: 1,
```

Comment:
Same here, we don't need to increment `numberOfObjects` here. It gets incremented in complete mpu api.